### ISSUE TITLE
Cannot open with keepass2android - however, opens fine on desktop with keepassxc

### ISSUE THREAD

**Issue by raghur** [PhilippC/keepass2android#468]
So this is the first time that I'm faced with this issue.

This evening, all of a sudden, kp2a wouldn't open my password database and there's a message box that the db is probably corrupted. First thing I did was check on my desktop to make sure that the file is openable. I'm on KDE neon and using keepassxc 2.3.3 on the desktop and am able to open the file.

My password db is synced with syncthing - so the next thing I did was to create a copy of the password db in the same folder on my desktop and wait for it to sync to my phone. On the phone, I used kp2a to try and open the copy. Of course, since this is the first time, I have to enter my password. However, the copy db doesn't open as well - same error message.

My set up:
Syncthing to sync between phone, desktop and private cloud servers.
KP2A version 1.06-pre1
Desktop - KDE Neon User edition, keepassxc appimage 2.3.3

**Comment by raghur** [comment]
Just updated to 1.06a and there's no difference.
I have compared md5 hashes of the file on the mobile and pc as well as copied the file over from the mobile separately and opened it.
I've now opened it on PC and disabled compression in db settings and syncing it to see if it helps. Update: it doesn't.
I've also tried downgrading to 1.05d (github) and that's not able to open the database as well.

The only recent thing that I recall that was off the normal was that I had a sync conflict and had to merge the database using keepassxc.

It's late here - but I guess I'll try opening the database with keepass (the original).
